web/modules/custom/popup/popup.routing.yml

```diff
@@ -1,7 +1,15 @@
 popup.settingslink:
-  path: '/users/settings'
+  path: "/users/settings"
   defaults:
-    _title: 'Setting'
+    _title: "Setting"
     _controller: '\Drupal\popup\Controller\SettingController::setting'
   requirements:
-    _permission: 'access content'
+    _permission: "access content"
```

Comment:
Why content access permissions, should be something related to user updates right?